### ISSUE TITLE
[New Rule] Use set_default in Python dictionaries

### ISSUE THREAD

**Issue by Skylion007** [MartinThoma/flake8-simplify#182]
## Explanation

Explain briefly why you think this makes the code simpler.

## Example

```python
# Bad
a = {"key": "value"}
if "key" not in a:
    a["key"] = "default_value"

# Good
a = {"key": "value"}
a.set_default("key", "default_value")
```

Faster, more efficient, removes an if statement.
